applications/content-drafting.md

```diff
@@ -0,0 +1,29 @@
+Machine translation allows to quickly draft content in natural language. Applications:
+
+# Mock-up localization
```

Comment:
For subtitles, let's use [H3 format](https://machinetranslate.org/style#headers).

Also, we default to [UK spelling](https://machinetranslate.org/style#default-to-wikipedia-style).

### Mock-up localisation